README.rst

```diff
@@ -115,17 +115,15 @@ Using an environment variable:
 
 .. code-block:: console
 
-    $ BLOSC_DIR=/usr/local     (or "set BLOSC_DIR=\blosc" on Win)
-    $ export BLOSC_DIR         (not needed on Win)
-    $ python setup.py build_clib
-    $ python setup.py build_ext --inplace
+    $ export USE_SYSTEM_BLOSC=1                 # or "set USE_SYSTEM_BLOSC=1" on Windows
+    $ export Blosc_ROOT=/usr/local/customprefix # If you installed Blosc into a custom location
```

Comment:
This mix of upper and lower case in `Blosc_ROOT` looks a bit inconsistent.  I suggest to use `BLOSC_ROOT`.

Reply:
Please consult the CMake documentation and reconsider: https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#standard-variable-names

Reply:
Ok.  I can see that CMake guys prefer the mixed case, that's ok.  However, I'd continue advocating for making the *environment* variable consistently all in upper case.  If you want, you can create a new variable inside CMake that is mixed case.  Sorry if I seem a bit picky about this, but I think taking this additional action would save some headaches to users.

Reply:
[The environment variable name is a direct consequence of the name of FindBlosc.cmake due to how CMake handles find_modules](https://cmake.org/cmake/help/latest/envvar/PackageName_ROOT.html#envvar:%3CPackageName%3E_ROOT). This line is just a hint in the README how to manipulate CMake for non-standard setups.

Reply:
Holly cow.  I definitely disagree with conventions that CMake team is making even for *environment* variables.  But if that is the naming hell where we are all headed, be it.

Reply:
I think you underestimate the power of how CMake handles it's variables. Envvars, cached vars, command-line flags, they are all related somehow and have a predetermined precedence.

Reply:
Yeah, maybe.  I just don't like users being presented with these conventions.  As said, they look weird to my eyes (but hopefully it is only me :-P).